doc/latex.rst

```diff
@@ -520,9 +520,19 @@ LaTeX boolean keys require *lowercase* ``true`` or ``false`` values.
 Spaces around the commas and equal signs are ignored, spaces inside LaTeX
 macros may be significant.
 
+``latexlistlabels``
+    Defaults to ``false``.  It means that enumerated list items will follow
+    the format (alphabetical or numerical) from the source.  Setting to
+    ``true`` means to let LaTeX document class or dedicated package decide
+    of the style of enumeration according to nesting level.
+
+    .. versionadded:: 2.1.3
+       Use this via raw latex directive for local modification.
+       Use rather :confval:`latex_use_latexlistlabels` for a global setting.
+
```

Comment:
I hesitated to document this (which is used anyhow by LaTeX writer for implementation of ``latex_use_latexlistlables`` confval), as it is of interest only for rare cases where one needs a local exception to global rule, and will be used only by LaTeX proficient people.

Reply:
Previous comment changes meaning with latest commit which dropped `latex_use_latexlistlabels`  config value in favour of keeping only the `sphinxsetup` TeX boolean interface.